github/ci/prow-deploy/files/jobs/kubevirt/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits.yaml

```diff
@@ -22,7 +22,7 @@ presubmits:
       nodeSelector:
         type: bare-metal-external
       containers:
-      - image: quay.io/kubevirtci/kubevirt-infra-bootstrap:v20230103-9f4e101
```

Comment:
IIRC you need to set the new presets also for podman to work. (@brianmcarey please keep me honest here):

```
      preset-docker-mirror-proxy: "true"
      preset-podman-in-container-enabled: "true"
      preset-podman-shared-images: "true"
```

Reply:
added `preset-podman-in-container-enabled: "true"`

thanks
i dont think we need the other
we use the previous one on podman images as well
(maybe worth to rename in the future)

Reply:
IIRC the latter one is used to mount the host path `/var/lib/shared-images` for podman to benefit from using the images that `shared-images-controller` updates regularly, no? @brianmcarey FYI

Reply:
maybe, if its true we will need to change all over btw

Reply:
Done (unless someone says otherwise)